### ISSUE TITLE
Feature requests 

### ISSUE THREAD

**Issue by Chickenshrimps** [rebelonion/Dantotsu#445]
Add a data saver option for manga and anime both. While opening certain anime/manga it takes quite much of my data to actually start playing episode/chapter 
Data saver option would be best for not only smooth browsing through mangas/anime section but also to download episodes/chapters without taking toouch of a data 
Pls look forward to it, 
Thank you

**Comment by sneazy-ibo** [comment]
The only thing I could think of for data saving is not loading any images and just displaying text instead. Regarding the second thing you mentioned it also means that you are going to give up on image and video quality. Also what you are requesting has been requested many times before already so please check if your idea is actually something new

**Comment by Chickenshrimps** [comment]
Yes some chapters does load properly and source error exception pops on my screen several times. Pls fix it in the next update

**Comment by itsmechinmoy** [comment]
> And source error exception pops on my screen several times. Pls fix it in the next update

We don't maintain source (ask aniyomi to fix it)

**Comment by sneazy-ibo** [comment]
> We don't maintain source (ask aniyomi to fix it)

bruh they won't fix it, it literally got dmca'ed 😭

**Comment by itsmechinmoy** [comment]
> bruh they won't fix it, it literally got dmca'ed 😭

suffer

**Comment by itsmechinmoy** [comment]
> Also what you are requesting has been requested many times before already so please check if your idea is actually something new

most suggestions here had already been suggested b4r in discord T_T
